### ISSUE TITLE
the question about the requestNumber

### ISSUE THREAD

**Issue by wukong1992** [kubeflow/katib#161]
I create a studyjobs, and it continue create pods, and I configure the requestNumber  "3", I didn't know the specfic implementation in the katib,

**Comment by wukong1992** [comment]
I didn't exactly know the meaning of requestNmber，the number of workers at the same time?

**Comment by YujiOshima** [comment]
@wukong1992 Yes.  The meaning of requestNmber is the maximum number of workers in parallel.

**Comment by richardsliu** [comment]
@YujiOshima I think in future versions of the katib API we should make this clearer. For example, "requestCount" and "requestNumber" are both present in the schema, and it is not obvious what they mean.

**Comment by jlewi** [comment]
Duplicate of #370
